clicommand/artifact_download.go

```diff
@@ -13,12 +13,27 @@ var DownloadHelpDescription = `Usage:
 
 Description:
 
-	 Downloads artifacts specified by <query> from Buildkite to <destination>
-	 directory on the local machine.
+   Downloads artifacts matching <query> from Buildkite to <destination>
+   directory on the local machine.
 
    Note: You need to ensure that your search query is surrounded by quotes if
-   using a wild card as the built-in shell path globbing will provide files,
-   which will break the download.
+   using a wild card as the built-in shell path globbing will expand the wild
+   card and break the query.
+
+   If the last path component of <destination> matches the first path component
+   of your <query>, the last component of <destination> is dropped from the
+   final path e.g. a query of 'app/logs/*' with a destination of 'foo/app' will
```

Comment:
This looks great! The only thing I'd change is to make this a full stop and start the example with For example instead of e.g. here and in the next para. But that's personal preference, feel free to ignore 😊

Reply:
Done!